src/chains/ethereum/ethereum/src/api.ts

```diff
@@ -949,7 +951,8 @@ export default class EthereumApi implements Api {
         options.miner.difficulty,
         parentHeader.totalDifficulty,
         blockchain.getMixHash(parentHeader.parentHash.toBuffer()),
-        0n // no baseFeePerGas for estimates
+        0n, // no baseFeePerGas for estimates
+        KECCAK256_RLP
```

Comment:
`KECCAK256_RLP` is the empty trie root: https://github.com/ethereumjs/merkle-patricia-tree/blob/master/src/baseTrie.ts#L56